.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please delete the .idea folder from your PR: It is for local development only and should not be commited. 
 Best add it to the .gitignore instead